docs/guide/terraform-guide.md

```diff
@@ -0,0 +1,181 @@
+- [Sematext Cloud Account](https://apps.sematext.com/ui/account/api)
+- Sematext Cloud API Access Token (from you cloud account).
+- Your choice of suitable [Sematext Cloud Plan ID](https://github.com/sematext/terraform-provider-sematext/blob/master/docs/guides/plans.md)
+
+### Environment
+
+Create the following environment variables:
+
+```sh
+export SEMATEXT_API_KEY="<your access key>"
+export SEMATEXT_REGION="<US or EU>"
+```
+
+### Create a Terraform HCL script
+
+In this simplistic case we create a file containing the following Terraform snippet. 
+
+
+```hcl
+terraform {
+  required_providers {
+    sematext = {
+      source = "sematext/sematext"
+      version = ">=0.1.3"
+    }
+  }
+}
+
+provider "sematext" {
+    sematext_region = "US"
+}
+
+resource "sematext_monitor_mongodb" "monitor_mongodb" {
+    name = "MongoDB Monitor Example"
+    billing_plan_id = 125 
+}
+```
+
+Save this to a file e.g. main.tf
+
+We will use this to create a monitoring app on Sematext Cloud, ready to receive, in this case MongoDB Metrics.
+Note this will just demonstrate a starting point. Usually you would combine other Providers and deploy suitable agent-collectors.
+
+<br>
+
+### Step 1 : Initialize a Deployment 
+
+```bash
+$ terraform init
+```
+
+This will download down the Terraform Provider ready for use.
+
+### Step 2 : Create a Deployment Plan
+
+```bash
+$ terraform plan
+```
+
+This will tell you the plan is to deploy a new sematext_monitor_mongodb.
+
+
+### Step 3 : Execute the Deployment
+
+```bash
+$ terraform apply
+```
+
+You now have a new monitoring app on Sematext Cloud, ready to receive, in this case MongoDB Metrics.
+
+### Step 4 : Modify the Plan
+
+We also want to be able to use 
+
+Append the following snippet to the bottom of your main.tf
+
+```hcl
+resource "sematext_monitor_elasticsearch" "monitor_elasticsearch" {
+    name = "Elasticsearch Monitor Example"
+    billing_plan_id = 12 
+}
+```
+
+### Step 5 : Re-Plan Deployment
+
+```bash
+$ terraform plan
+```
+
+This will tell you the plan is to deploy an additional new sematext_monitor_elasticsearch.
+
+
+### Step 6 : Execute Your Deployment
+
+```bash
+$ terraform execute
+```
+ You will now have an additional Sematext Cloud Elasticsearch App ready to receive metrics.
+
+
+### Step 7 : Clean up
+
+Once you are finished taking a look and you want to retire the apps just 
+ 
+```bash
+$ terraform destroy
+```
+
+Note this sets the apps to a retire status. They will be removed after 30 days.
+
+Refer to our [examples repo](https://github.com/sematext/terraform-examples) for more realistic examples.
+
+<br><br>
+## Further links to know more
+
+- [Terraform Registry Sematext Provider](https://registry.terraform.io/providers/sematext/sematext/latest)
+- [Sematext Provider Github](https://github.com/sematext/terraform-provider-sematext)
+- [Sematext Manual Integrations Guide](https://sematext.com/docs/guide/integrations-guide/)
```

Comment:
@adnanrahic is this Guide a keeper?  If not, we don't need to link to it here.

Reply:
Nope, we plan on removing it. Let's link to https://sematext.com/docs/integration/ instead.

Reply:
Link changed.